files/es/web/api/navigator/getusermedia/index.md

```diff
@@ -79,7 +79,7 @@ function(localMediaStream) {
 
 ### errorCallback
 
-La función `getUserMedia `llama a la función indicada en el errorCallback con un `código` como argumento. Los códigos de error se describen a continuación:
+La función `getUserMedia`llama a la función indicada en el errorCallback con un `código` como argumento. Los códigos de error se describen a continuación:
```

Comment:
Need to watch out for these ones, where Markdownlint trims space inside the code span, but it's missing from the outside

Reply:
Thank you so much @nschonni did you check these manually or have an script?

Reply:
Just manually unfortunately

Reply:
Thank you so much, I'm going to start reviewing the whole PR now as the past PRs